custom_components/control4/light.py

```diff
@@ -37,7 +37,7 @@ async def async_setup_entry(
 
     for item in items_of_category:
         try:
-            if item["type"] == CONTROL4_ENTITY_TYPE and item["id"]:
```

Comment:
What does this do? Why do we need to check for "fan" inside the light integration?

Reply:
The fan controllers get detected as lights, that's what made me tackle fan controllers.   They would add a bunch of errors to my logs.

Reply:
Ah that makes sense, could you add a short comment in the code here just explaining that so it doesn't get accidentally removed in the future?